docs/configuration.rst

```diff
@@ -168,3 +168,17 @@ Here is a more complex example:
      ignore: |
        *.ignore-trailing-spaces.yaml
        ascii-art/*
+
+YAML file extensions
+--------------
+
+To configure what yamllint should consider as yaml file, set `yaml-files` configuration option.
```

Comment:
Perfect! Just a correction: it should be file**s**, e.g.:

> ... as YAML files

Reply:
@adrienverge thanks for the hints, fixed